sklearn/model_selection/_split.py

```diff
@@ -1467,7 +1467,7 @@ def get_n_splits(self, X=None, y=None, groups=None):
 class _CVIterableWrapper(BaseCrossValidator):
     """Wrapper class for old style cv objects and iterables."""
     def __init__(self, cv):
-        self.cv = cv
+        self.cv = list(cv)
```

Comment:
(Just to note that I've added the 2nd test just to ensure this change has effect and to fail if reverted later...)